doc/releasing.md

```diff
@@ -0,0 +1,50 @@
+# Release process
+
+Target audience: crate maintainers who wish to release `embedded-graphics` or `embedded-graphics-core`.
+
+> Please take a cautious approach to this. If any step doesn't feel right or doesn't succeed smoothly, stop and rectify any issues before continuing.
+
+## On Github
+
+- Check that all desired PRs are merged and all desired issues are closed/resolved.
+- Check that the latest master build passed in CircleCI.
+
+## On your local machine
+
+- `cd` to the repository root
+- Use the crate MSRV of 1.40 by running `rustup override set 1.40`
+- Check that `just` and `cargo-release` are installed and available in `$PATH`.
+  - `just --version`
+  - `cargo release --version`
+- Ensure you have the latest changes with `git checkout master` and `git pull`
```

Comment:
Should we use the new `switch` command instead of `checkout`?
```suggestion
- Ensure you have the latest changes with `git switch master` and `git pull`
```

Reply:
TIL that's a command! My muscle memory has always used `checkout`.